#! /bin/sh /usr/share/dpatch/dpatch-run
## 404_undef_USE_PAM.dpatch by Nicolas FRANCOIS <nicolas.francois@centraliens.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not use PAM for chage, chpasswd, groupadd, groupdel, groupmod
## DP: newusers, useradd, userdel, usermod
## DP: 
## DP: I don't know if it must be kept.
## DP: see also #283961, #162181, #162199, #162228.

@DPATCH@
diff -urNad shadow-4.0.3/src/chage.c /tmp/dpep.a88PWo/shadow-4.0.3/src/chage.c
--- shadow-4.0.3/src/chage.c	2005-04-16 21:23:27.935795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/chage.c	2005-04-16 21:25:07.565795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: chage.c,v 1.27 2002/01/05 15:41:43 kloczek Exp $")
 #include <sys/types.h>
diff -urNad shadow-4.0.3/src/chpasswd.c /tmp/dpep.a88PWo/shadow-4.0.3/src/chpasswd.c
--- shadow-4.0.3/src/chpasswd.c	2005-04-16 21:23:27.935795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/chpasswd.c	2005-04-16 21:25:07.565795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: chpasswd.c,v 1.14 2002/01/05 15:41:43 kloczek Exp $")
 #include <stdio.h>
diff -urNad shadow-4.0.3/src/groupadd.c /tmp/dpep.a88PWo/shadow-4.0.3/src/groupadd.c
--- shadow-4.0.3/src/groupadd.c	2005-04-16 21:23:27.935795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/groupadd.c	2005-04-16 21:25:07.565795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: groupadd.c,v 1.22 2002/01/06 14:09:07 kloczek Exp $")
 #include <sys/types.h>
diff -urNad shadow-4.0.3/src/groupdel.c /tmp/dpep.a88PWo/shadow-4.0.3/src/groupdel.c
--- shadow-4.0.3/src/groupdel.c	2005-04-16 21:23:27.935795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/groupdel.c	2005-04-16 21:25:07.565795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID(PKG_VER "$Id: groupdel.c,v 1.14 2000/10/09 19:02:20 kloczek Exp $")
 
diff -urNad shadow-4.0.3/src/groupmod.c /tmp/dpep.a88PWo/shadow-4.0.3/src/groupmod.c
--- shadow-4.0.3/src/groupmod.c	2005-04-16 21:23:27.935795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/groupmod.c	2005-04-16 21:25:07.565795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: groupmod.c,v 1.19 2002/01/05 15:41:43 kloczek Exp $")
 #include <sys/types.h>
diff -urNad shadow-4.0.3/src/newusers.c /tmp/dpep.a88PWo/shadow-4.0.3/src/newusers.c
--- shadow-4.0.3/src/newusers.c	2005-04-16 21:23:27.945795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/newusers.c	2005-04-16 21:25:07.605795000 +0200
@@ -35,6 +35,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: newusers.c,v 1.15 2002/01/05 15:41:43 kloczek Exp $")
 #include <sys/types.h>
diff -urNad shadow-4.0.3/src/useradd.c /tmp/dpep.a88PWo/shadow-4.0.3/src/useradd.c
--- shadow-4.0.3/src/useradd.c	2005-04-16 21:23:27.955795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/useradd.c	2005-04-16 21:25:07.665795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: useradd.c,v 1.32 2002/01/10 13:01:28 kloczek Exp $")
 #include "prototypes.h"
diff -urNad shadow-4.0.3/src/userdel.c /tmp/dpep.a88PWo/shadow-4.0.3/src/userdel.c
--- shadow-4.0.3/src/userdel.c	2005-04-16 21:23:27.955795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/userdel.c	2005-04-16 21:25:07.725795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: userdel.c,v 1.21 2002/01/05 15:41:44 kloczek Exp $")
 #include <sys/stat.h>
diff -urNad shadow-4.0.3/src/usermod.c /tmp/dpep.a88PWo/shadow-4.0.3/src/usermod.c
--- shadow-4.0.3/src/usermod.c	2005-04-16 21:23:27.955795000 +0200
+++ /tmp/dpep.a88PWo/shadow-4.0.3/src/usermod.c	2005-04-16 21:25:07.775795000 +0200
@@ -29,6 +29,8 @@
 
 #include <config.h>
 
+#undef USE_PAM
+
 #include "rcsid.h"
 RCSID (PKG_VER "$Id: usermod.c,v 1.25 2002/01/05 15:41:44 kloczek Exp $")
 #include <sys/types.h>
